pages/mesosphere/dcos/services/elastic/3.1.0-7.4.1/release-notes/index.md

```diff
@@ -8,6 +8,26 @@ model: /mesosphere/dcos/services/elastic/data.yml
 render: mustache
 ---
 
+# Release notes for Elastic Service version 3.1.0-7.4.1
+
+## Upgrades
+
+-   Upgraded Elasticsearch and Kibana to [7.4.1](https://www.elastic.co/guide/en/elasticsearch/reference/7.4/release-notes-7.4.1.html)
+-   Upgraded SDK version from `0.56.3` to `0.57.3`. ([#70](https://github.com/mesosphere/dcos-elastic-service/pull/70))
```

Comment:
Given the bug fix in SDK `0.57.3` which fixes the possiblity of a data-loss, can you add the `pod replace` notice which is found in the SDK release notes here.